client/stylesheets/_toCat.scss

```diff
@@ -4832,16 +4834,26 @@ On click the images will be displayed at normal size to complete the effect
 	position:absolute;
 	left:0;
 	right:0;
+	border: 2px solid #f39d00;
+	margin-top: -2px;
 }
 
 .optionsClick ul .clicker {
 	position:relative;
-	background:#2284B5;
+	background:orange;
```

Comment:
Color `orange` should be written in hexadecimal form as `#ffa500`<br>Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space